src/libpgmoneta/value.c

```diff
@@ -456,14 +457,24 @@ string_to_string_cb(uintptr_t data, int32_t format, char* tag, int indent)
    {
       if (format == FORMAT_JSON || format == FORMAT_JSON_COMPACT)
       {
-         snprintf(buf, MISC_LENGTH, "\"%s\"", str);
+         for (int i = 0; i < strlen(str); i++)
```

Comment:
`strlen()` is O(n) and you are calling it every loop, instead you should store the length as variable somewhere else.